docs/logs/discovery/tailing-log-files.md

```diff
@@ -0,0 +1,54 @@
+title: Shipping Log Files
+description: Shipping Log File Sources
+
+Enabling automatic logs shipping is accomplished by triggering the "Set up" button that opens the flyout for configuring log file shipping.
+
+![Setup Button](images/setup.png)
```

Comment:
I think it is more valuable to people not intimately familiar with the UI to show where in the UI they should look to find something than the zoomed-in and cropped image.  A "big picture" screenshot annotated with an arrow pointing to the relevant bit(s) works better, IMHO.

Reply:
Adjusted. 

![log-files-docs](https://user-images.githubusercontent.com/11174375/100226307-d6d97800-2f1f-11eb-8147-465be182b8bd.png)